Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py

```diff
@@ -1440,15 +1447,18 @@ def find_folders(client: EWSClient, target_mailbox=None):
     :return: Output tuple
     """
     account = client.get_account(target_mailbox)
-    root = account.root
+    root_collection = FolderCollection(account=account, folders=[account.root])
+    root_folder = account.root
+
     if client.is_public_folder:
-        root = account.public_folders_root
+        root_folder = account.public_folders_root
```

Comment:
Wouldn't this fail like here: https://github.com/demisto/content/blob/ea54b979cd8af1c67338535f5654218874482f60/Packs/MicrosoftExchangeOnline/Integrations/EWSO365/EWSO365.py#L347

Reply:
Dan did this on the next line, so i think that line shouldnt fail. didnt check yet
`            inbox_folders = FolderCollection(account=account, folders=[account.public_folders_root])
`